.gitignore

```diff
@@ -165,6 +165,8 @@ compilation-stats.json
 !pkg/services/featuremgmt/toggles_gen.go
 !pkg/coremodel/**/*_gen.go
 !pkg/framework/**/*_gen.go
+!pkg/plugins/pfs/**/*_gen.go
```

Comment:
still needed?

Reply:
this one is, there's still the generated "registry"